src/cloud-api-adaptor/podvm/Makefile

```diff
@@ -82,15 +84,21 @@ $(IMAGE_FILE): $(BINARIES) $(FILES) setopts
 	mkdir -p toupload
 	if [ "${SE_BOOT}" = "1" ] && [ "${ARCH}" = "s390x" ]; then \
 		qemu-img create -f qcow2 "se-${IMAGE_FILE}" 100G; \
+		# Temporary workaround for installing cryptsetup on RHEL 9.4 and below s390x base images for enabling se \
+		# Due to issue: https://gitlab.com/qemu-project/qemu/-/issues/2054 \
+		# Remove this if using the latest QEMU version (v9.0.0) \
+		if [ "${PODVM_DISTRO}" = "rhel" ]; then \
+			yum install -y cryptsetup; \
+			cp /usr/sbin/cryptsetup ./files/usr/bin/; \
+		fi \
```

Comment:
should be
```
		fi; \
```

Reply:
@liudalibj @Saripalli-lavanya is this still relevant ?

Reply:
I think no, now we were able to build image with out changes.